src/Controller/StaticPagesController.php

```diff
@@ -38,4 +38,14 @@ public function indexCgu(): Response
     {
         return $this->render('static/cgu.html.twig');
     }
+
+    /**
+     * This method is used to display the partner page
+     * @Route("/partners", name="partners")
+     * @return Response
+     */
+    public function indexPartners(): Response
+    {
+        return $this->render('/partners.html.twig');
```

Comment:
You should not have final views directly under the templates directory
This one should probably be in the static directory because it extends the static layout